conf/test.config

```diff
@@ -16,14 +16,16 @@ params {
 
     // Limit resources so that this can run on GitHub Actions
     max_cpus   = 2
-    max_memory = '6.GB'
+    max_memory = '12.GB'
```

Comment:
why 12?
Is this working well on GHA?

Reply:
Yes, IIRC they increased resources, and I think I needed it here.